packages/test-utils/src/wrapper.js

```diff
@@ -766,7 +784,8 @@ export default class Wrapper implements BaseWrapper {
    */
   setSelected (): void {
     const tagName = this.element.tagName
-    const type = this.attributes().type
+    const attributes = this.attributes()
+    const type = typeof attributes === 'object' ? attributes.type : ''
```

Comment:
Was this to fix flow errors? If so you can just add `// $FlowIgnore` above the original line. We know it's never going to return a string, so there's no need to guard against it.

We're actually going to replace flow with TypeScript soon, which would fix this issue. It's better to add flow ignores now rather than add extra code to satisfy flow.